examples/spin-llm/app.py

```diff
@@ -1,7 +1,8 @@
-from spin_sdk.http import IncomingHandler, Request, Response
+from spin_sdk import http   
```

Comment:
Maybe combine this import with the last one, e.g. `from spin_sdk import http, llm`?  And likewise for the other examples.